hydrus/samples/hydra_doc_sample.py

```diff
@@ -555,35 +555,35 @@
                     "writeonly": "false"
                 },
                 {
-                    "hydra:description": "The UserCollection collection",
-                    "hydra:title": "usercollection",
+                    "hydra:description": "The OrderCollection collection",
+                    "hydra:title": "ordercollection",
                     "property": {
-                        "@id": "vocab:EntryPoint//user",
+                        "@id": "vocab:EntryPoint/store/order",
                         "@type": "hydra:Link",
-                        "description": "The UserCollection collection",
+                        "description": "The OrderCollection collection",
                         "domain": "vocab:EntryPoint",
-                        "label": "UserCollection",
-                        "range": "vocab:UserCollection",
+                        "label": "OrderCollection",
+                        "range": "vocab:OrderCollection",
                         "supportedOperation": [
                             {
-                                "@id": "_:_:user_collection_retrieve",
+                                "@id": "_:_:order_collection_retrieve",
```

Comment:
This is still wrong. It should be `_:user_collection_retrieve`

Reply:
actually @xadahiya i also saw this abnormality but ignored since there were multiple instances of this [here](https://github.com/HTTP-APIs/hydra-flock-central-controller/blob/master/flock_controller/api_docs/doc.py#L1143)

Reply:
Okay, create an issue for this. Merging PR